_dashboards/management/advanced-settings.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+title: Advanced settings
+parent: Dashboards Management
+nav_order: 20
```

Comment:
@joshuarrrr What are the required permissions?

Reply:
@davidlago Can someone from your team provide some specifics on the roles/permissions that map to actually making changes to the advanced settings? My understanding is that any user with read settings still has access to view the settings and page, but is simply prevented from saving changes without the right permissions.

Reply:
Advanced settings are saved objects much like visualizations, index patterns and dasbhoards. The access is granted the same way users get access to the tenant where the advanced settings "live": https://opensearch.org/docs/latest/security/multi-tenancy/multi-tenancy-config/#give-roles-access-to-tenants

cc: @peternied to check me here and make sure I'm not mis-speaking.

Reply:
👍 

Restating what has already been mentioned; for the dashboards objects, the items stored in the `.kibana` index, there are only read and read+write permissions as is linked out to on the [[give-roles-access-to-tenants]](https://opensearch.org/docs/latest/security/multi-tenancy/multi-tenancy-config/#give-roles-access-to-tenants) link.